common/scala/src/main/scala/whisk/core/containerpool/Container.scala

```diff
@@ -142,12 +142,16 @@ trait Container {
     callContainer("/run", body, timeout, retry = false)
       .andThen { // never fails
         case Success(r: RunResult) =>
```

Comment:
Did you consider adding a similar behaviour for the `/init` call as well? I see nearly the same code in https://github.com/apache/incubator-openwhisk/blob/master/common/scala/src/main/scala/whisk/core/containerpool/Container.scala#L102 

or maybe even extracting it into a separate method would help here...